a11y-input/test/ally-input.test.js

```diff
@@ -29,10 +29,19 @@ describe('a11y input', () => {
     `));
 
     // ref: @open-wc chai plugin https://open-wc.org/testing/semantic-dom-diff.html
-    expect(el).shadowDom.to.equal(`
-      <slot name="label"></slot>
-      <slot name="input"></slot>
-    `);
+    /**
+     * manual snapshot testing...
+     *
+     */
+    // expect(el).shadowDom.to.equal(`
+    //   <slot name="label"></slot>
+    //   <slot name="input"></slot>
+    // `);
+
+    /**
+     * semi/automatic snapshots using the `Karma` plugin
+     */
+    expect(el).shadowDom.to.equalSnapshot();
```

Comment:
1. First run will generate a `snapshot`.
2. Second and following runs will use the cached (generated) `snapshot` in `__snapshots`, unless explicitly updated with:

```
$ yarn test:update-snapshots
```